multichain-testing/test/send-anywhere.test.ts

```diff
@@ -0,0 +1,142 @@
+import anyTest from '@endo/ses-ava/prepare-endo.js';
+import type { TestFn } from 'ava';
+import { makeDoOffer } from '../tools/e2e-tools.js';
+import {
+  commonSetup,
+  SetupContextWithWallets,
+  chainConfig,
+} from './support.js';
+import { createWallet } from '../tools/wallet.js';
+import { AmountMath } from '@agoric/ertp';
+import { makeQueryClient } from '../tools/query.js';
+import type { Amount } from '@agoric/ertp/src/types.js';
+
+const test = anyTest as TestFn<SetupContextWithWallets>;
+
+const chains = ['osmosis', 'cosmoshub'];
+const scenarios = Array(2)
+  .fill(undefined)
+  .flatMap((_, i) => chains.map(c => [c, i])) as [string, number][];
+const accounts = scenarios.map(args => args.join(''));
```

Comment:
it took me some minutes to make sense of this (the logic and where the values are used). Consider:
```suggestion
const accounts = ['osmosis', 'cosmoshub'].map((name, i) => `${name}${i}`);
```

And rather than data-driving with `scenarios`, just explicitly call each one.

Reply:
We're testing with 4 accounts, so I explicitly wrote them out. (the above would just return 2). Acknowledge the readability feedback and implemented your suggestions 🙏